webviz_config/generic_plugins/_banner_image.py

```diff
@@ -39,16 +40,23 @@ def __init__(
         self.color = color
         self.shadow = shadow
         self.height = height
+        self.title_position = title_position
 
         self.image_url = WEBVIZ_ASSETS.add(image)
 
     @property
     def layout(self) -> html.Div:
 
+        if self.title_position == "top":
+            self.title_position = "start"
+        elif self.title_position == "bottom":
+            self.title_position = "end"
+
```

Comment:
In order to avoid overriding the `self.title_position` property set in `__init__`, maybe we can translate it when requested. I'm thinking e.g. something along the lines of a constant lookup dictionary
```python
TITLE_POSITIONS: Dict[str, str] = {
    "top": "start",
    "center": "center",
   "bottom": "end"
}
```
and then 
```python
        style = {
            "color": self.color,
            "backgroundImage": f"url({self.image_url})",
            "height": f"{self.height}px",
            "align-items": TITLE_POSITIONS[self.title_position],
        }
```
which also gives the possibility of catching `KeyError` with a user friendly error message regarding invalid position 🙂

Reply:
Okay, sounds good! I did not think about that. I will make the changes today and add an entry on the `CHANGELOG.md`.